src/codebook/controls/filters/update.js

```diff
@@ -0,0 +1,75 @@
+/*------------------------------------------------------------------------------------------------\
```

Comment:
yep - still need the full enter/exit flow (those bits just be ignored when called from init). Will review updates later this afternoon.